architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -291,6 +372,19 @@ the xref:image-stream-object-definition[example above] would be
 ----
 ====
 
+
+[[image-stream-triggers]]
+=== Image Stream Triggers
+
+Image change triggers allow your build or deployment to be automatically invoked when a new version of an upstream image is available.
```

Comment:
@bparees I think we need a piece of doc about the generic trigger. Both @mfojtik and @tnozicka where asking me about it earlier today.